value-annotations/src/org/immutables/value/Value.java

```diff
@@ -916,6 +916,14 @@
      */
     ImplementationVisibility visibility() default ImplementationVisibility.SAME;
 
+    /**
+     * Specify the mode in which visibility of generated value type is derived from abstract value
+     * type. It is a good idea to not specify such attributes inline with immutable values, but
+     * rather create style annotation (@see Style). Specifying this will override {@link #visibility()}.
```

Comment:
Need to expand documentation with the note about Javac warning (and maybe error-prone's one) at least briefly

> It is a good idea to not specify such attributes inline with immutable values, but rather create style annotation (@see Style)

This sorta tries to explain this, but without context (of warnings) it's not explaining it well why these `*String()` exist